docs/antrea-ipam.md

```diff
@@ -20,6 +20,20 @@ Note that running Antrea NodeIPAM while NodeIPAMController runs within
 kube-controller-manager would cause conflicts and result in an unstable
 behavior.
 
+### Enable NodeIPAM feature gate
```

Comment:
should it be put together with other configuration of NodeIPAM like the example of AntreaIPAM

Reply:
Done, move it to `Configuration` section and refined the description.